File: Easy/Remove all duplicates from a given string.py
#User function Template for python3
class Solution:

	
	def removeDuplicates(self, str):
	    # code here
	    m = {}
	    ans = ""
	    
	    for c in str:
	        if c not in m:
	            m[c] = 1
	            ans += c
	        
        return ans

#{ 
 # Driver Code Starts
#Initial Template for Python 3





if __name__ == '__main__':
    tc = int(input())
    while tc > 0:
        str = input().strip()
        ob = Solution()
        ans = ob.removeDuplicates(str)
        print(ans)
        tc -= 1

# } Driver Code Ends

File: Easy/Remove all duplicates from a given string.cpp
//{ Driver Code Starts
#include <bits/stdc++.h>

using namespace std;


// } Driver Code Ends
//User function template for C++
class Solution{
public:
	string removeDuplicates(string str)
	{
	    // code here
	    int n = str.length();
	    int mappa[256] = {};
	    
	    string ans = "";
	    
	    for (int i=0; i<n; i++)
	    {
	        if (mappa[str[i]] == 0) ans += str[i];
	        mappa[str[i]]++;
	    }
	    
	    return ans;
	    
	}
};

//{ Driver Code Starts.

int main() {
    int t;
    cin >> t;
    while (t--) {
        string str;
        cin >> str;
        Solution ob;
        auto ans = ob.removeDuplicates(str);

        cout << ans << "\n";
    }
    return 0;
}
// } Driver Code Ends